docs/user_guide/gui_workflow.rst

```diff
@@ -56,19 +59,21 @@ only after you know the API.
    #. Click once on the qubit.
    #. Click ``Shift+F2``
    #. Double-click on the qubit.
-   #. In the new *Object Properties* window locate a tab *PCell
+   #. In the new *Instance Properties* window locate a tab *PCell
       parameters*.
-   #. Scroll down in the list of parameters to locate *Coupler lengths*.
-   #. Change the value in the text box to ``100,160,0``.
+   #. Scroll down in the list of parameters to locate *Include coupler between qubits*.
+   #. Click on the box.
    #. Click ``Ok``.
 
 #. **Create a new qubit.**
 
    #. Press ``F2`` to see the whole top cell.
    #. From the *Tools* panel select *Instance*.
-   #. In the new *Object Editor Options* window locate *Library*.
-   #. Using the drop-down menu select *Element Library*.
-   #. Click on a button ``...`` a little bit to the left from the
+   #. Double-click on the qubit.
```

Comment:
This line should be removed and the previous line should be changed to say `toolbar` instead of `*Tools* panel`.

Reply:
@jkotilahti Hi, I am not so much which line you are saying should be removed do you mean the ```#. Double-click on the qubit```

Reply:
Yes, removing the `#. Double-click on the qubit` line is what I mean.